package.json

```diff
@@ -18,6 +18,7 @@
     "filestack-js": "^1.10.0",
     "git": "^0.1.5",
     "github-buttons": "git+https://github.com/ntkme/github-buttons.git",
+    "hotkeys-js": "^3.6.11",
```

Comment:
Not sure a new dependency is required for this.

```js
document.addEventListener('keydown', (e) => {
  if (e.metaKey && e.code === 'KeyK') { 
    console.log('Command + K pressed'); 
  }
});
```

Reply:
I tried that and it didn't work

Reply:
Hmm... I wrote that and tested in a browser without issue. 

Maybe we have another js dependency that's cancelling the default event behavior. My guess is the lib you added must be opting to capture the event and handle it before passing off to other handlers.

Reply:
See the `useCapture` argument here: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Reply:
I got it working :D